src/server/api/routes/events.router.js

```diff
@@ -0,0 +1,177 @@
+/* The API routing for events */
+
+const express = require('express');
+
+const router = express.Router({ mergeParams: true });
+
+// controllers
+const eventsController = require('../controllers/events.controller');
+
+/**
+ * @swagger
+ * /events:
+ *  get:
+ *    summary: Get all events
+ *    description:
+ *      Will return all events.
+ *    produces: application/json
+ *    responses:
+ *      200:
+ *        description: Successful request
+ *      5XX:
+ *        description: Unexpected error.
+ */
+router.get('/', (req, res, next) => {
+  eventsController
+    .getEvents()
+    .then((result) => res.json(result))
+    .catch(next);
+});
+
+/**
+ * @swagger
+ * /events/{ID}:
+ *  get:
+ *    summary: Get events by ID
+ *    description:
+ *      Will return single event with a matching ID.
+ *    produces: application/json
+ *    parameters:
+ *     - in: path
+ *       name: ID
+ *       schema:
+ *         type: integer
+ *         required: true
+ *         description: The ID of the event to get
+ *
+ *    responses:
+ *      200:
+ *        description: Successful request
+ *      5XX:
+ *        description: Unexpected error.
+ */
+router.get('/:id', (req, res, next) => {
+  eventsController
+    .geteventById(req.params.id)
+    .then((result) => res.json(result))
+    .catch(next);
+});
+
+/**
+ * @swagger
+ * /events:
+ *  post:
+ *    summary: Create a event
+ *    description:
+ *      Will create a event.
+ *    produces: application/json
+ *    parameters:
+ *      - in: body
+ *        name: event
+ *        description: The event to create.
+ *        schema:
+ *          type: object
+ *          required:
+ *            - event_type
+ *            - event_date
+ *            - venue
+ *             - maxParticipants
+ *          properties:
+ *            event_type:
+ *              type: enum
+ *            event_date:
+ *              type: string
+ *              format: date-time
+ *            venue:
+ *              type: string
+ *
+ *            maxParticipants:
+ *              type: int
+ *    responses:
+ *      201:
+ *        description: event created
+ *      5XX:
+ *        description: Unexpected error.
+ */
+router.post('/', (req, res) => {
```

Comment:
If you added these routes, why didn't you implement them? Did you test if they work?

Reply:
You still haven't removed these. They are breaking your code.